### PR TITLE
Added possibility to link external program pages

### LINKED ISSUE #598: Allow CMS user to direct program page to their own web site
Some programs (like SCM) may already have their own micromasters site. We should allow CMS users to specify a URL for their program pages instead of using our program pages.

### PR DESCRIPTION
**Pull request by giocalitri**
#### What are the relevant tickets?
closes #598 

#### What's this PR do?
allows the CMS users to specify an external link for the program page. The internal program page will not be linked from the homepage

#### Where should the reviewer start?
`cms/models.py`, then `cms/templates/cms/home_page.html`

#### How should this be manually tested?

- Add an URL for the program page in the CMS
- Verify that the program in the homepage links such URL

### CODE REVIEW

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -70,7 +70,11 @@
               {% for program in programs %}
               <li class="col-sm-6">
                 {% if program.programpage %}
-                  <a href="{{ program.programpage.url }}" class="program-link">
+                	{% if program.programpage.external_program_page_url %}
+                	<a href="{{ program.programpage.external_program_page_url }}" class="program-link" target="_blank">
```
Comment (noisecapella) [inline cms/templates/cms/home_page.html]: This should not have the `target="_blank"` according to [this recent comment](https://github.com/mitodl/micromasters/issues/598#issuecomment-227875534)

**Comment by noisecapella** [comment]
Functionality looks good

File: cms/templates/cms/home_page.html (hunk 1 above)
Comment (giocalitri) [inline cms/templates/cms/home_page.html]: Yep: comment came after my push! Will fix

**Comment by giocalitri** [comment]
updated

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**